crash_test.mk

```diff
@@ -8,7 +8,7 @@ DB_STRESS_CMD?=./db_stress
 include common.mk
 
 CRASHTEST_MAKE=$(MAKE) -f crash_test.mk
-CRASHTEST_PY=$(PYTHON) -u tools/db_crashtest.py --stress_cmd=$(DB_STRESS_CMD)
+CRASHTEST_PY=$(PYTHON) -u tools/db_crashtest.py --stress_cmd=$(DB_STRESS_CMD) --cleanup_cmd=$(DB_CLEANUP_CMD)
```

Comment:
Perhaps `'$(DB_CLEANUP_CMD)'` so that you don't have to do `make DB_CLEANUP_CMD='"rm -rf /blah"'`?